SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -542,7 +544,17 @@ void ActorSystem::HandleActorAuthority(const Worker_EntityId EntityId, const Wor
 						if (GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries
 							&& GetDefault<USpatialGDKSettings>()->bRefreshClientInterestOnHandover)
```

Comment:
Feels like this option should be removed now. We need to refresh on handover so we can repopulate the list?

Reply:
Yeah I agree, doesn't seem valid for this ever to be false now